prow/plugins/lgtm/lgtm.go

```diff
@@ -23,8 +23,8 @@ import (
 	"strings"
 
 	"github.com/sirupsen/logrus"
-	"k8s.io/apimachinery/pkg/util/sets"
 
+	"k8s.io/apimachinery/pkg/util/sets"
```

Comment:
k8s.io/apimachinery != k8s.io/test-infra. Please put it in the non-built-in, non-local repo block where it was previously

Reply:
Really good point, and thanks a lot for spotting this. I correct in this PR: https://github.com/kubernetes/test-infra/pull/12851 as my last PR was merged before I got home.